src/locales/en.json

```
@@ -344,6 +344,26 @@
         "Atomic": "Bug corrected in the periodic table",
         "Gutter": "Gutter corrected in the python application"
       }
+      "1-0-1": {
+        "XNT" : "XNT Cyclique (Plusieurs appuis sur XNT changent le caractère inséré entre X, N, T et θ)",
+        "NewPol" : "Changed font",
+        "TxtItal" : "Italicized text in some places such as function definitions in the functions application and texts and comments in the Python editor",
+        "TxtTool" : "Larger text in toolboxes with automatic scrolling when text is too long",
+        "LastApp" : "Shortcut Shift + Ans to go to the last used application",
+        "CorrPix" : "Fixed pixel shift on some icons in some themes",
+        "Dim" : "Dimension in additional results when a unit is used",
+        "PyMaJ" : "MicroPython update from 1.17 to 1.19.1",
+        "PyCrash" : "Fixed crash when opening scripts",
+        "PyItal" : "Italicized text in strings",
+        "AddUlab" : "Added `ulab.utils`.",
+        "PyLightControl" : "Control of brightness via `set_brightness` and `get_brightness`.",
+        "AddStat" : "Geometric mean, Harmonic mean and Mode in the Stats tab",
+        "AddFunctionLiseuse" : "Additional functions in TRU (sums, products, integrals and binomial coefficients)",
+        "AddCaracURT" : "Extra characters in URT (``f` and ``i`)",
+        "CorrBrownColor" : "Brown color correction in URT",
+        "CorrErrorBack" : "Fix syntax error when reverting",
+        "BlockHardware" : "Blocking of hardwares tests in exam mode",
```

Can you rename to `BlockHardwareTests` ?
```suggestion
        "BlockHardware" : "Blocking of hardwares tests in exam mode"
```